modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -1520,6 +1520,52 @@ A list of supported HTTP authentication mechanisms.
 
 ---
 
+=== iceberg_catalog_base_location
+
+Base path for the cloud object storage-backed Iceberg catalog. After Iceberg is enabled, do not change this value.
```

Comment:
@Deflaimun can we find out if we mean to say both "cloud" and "object"..? I think @andrwng might be able to help

Reply:
@andrwng on docs we usually side with object storage. Can I make this change here  ?